src/main/java/com/github/xetorthio/jedisque/BinaryJedisque.java

```diff
@@ -6,19 +6,27 @@
 
 import java.net.URI;
 import java.net.URISyntaxException;
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
 
 public class BinaryJedisque extends redis.clients.jedis.Connection {
 	static final private int DISQUE_PORT = 7711;
 	private final List<URI> uris = new ArrayList<URI>();
 	private Random randomGenerator = new Random();
+	private int timeout = 2000;
```

Comment:
I think this is the connection timeout value which will be changed by construction method 

```
    public BinaryJedisque(int timeout) {
        this();
        this.timeout = timeout;
    }
``` 

So, It can not be final .  static also useless, because every BinaryJedisque has it's own timeout value